insights/specs/default.py

```diff
@@ -678,11 +678,24 @@ def is_satellite_server(broker):
             return True
         raise SkipComponent
 
+    @datasource(CapsuleVersion)
```

Comment:
@huali027 please change this decorator to include `HostContext` like this: `@datasource(CapsuleVersion, HostContext)`.  See #2908 and #2911 for more information.

Reply:
@bfahr, sure, I'll update it.